src/control/path_follower/path_follower/node_pure_pursuit.py

```diff
@@ -161,24 +169,42 @@ def path_callback(self, spline_path_msg: PathStamped):
 
             self.img_initialised = True
 
-    def callback(self, msg: PoseWithCovarianceStamped):
+    # def callback(self, msg: PoseWithCovarianceStamped):
+    def timer_callback(self):
+        """
+        Listens to odom->base_footprint transform and updates the state.
+        Solution is to get the delta and add it to the previous state
+        and subtract the delta from the previous map->odom transform.
+        """
         # Only start once the path has been recieved, it's a following lap, and we are ready to drive
         if not self.following or not self.driving or self.path.size == 0:
+            self.started = False
+            return
+
+        if not self.started:
+            self.started = True
+            self.get_logger().info("Starting pure pursuit")
+
+        try:
+            odom_to_base = self.tf_buffer.lookup_transform("odom", "base_footprint", rclpy.time.Time())
```

Comment:
Should be track to base (have tested, does work)
track is the updated position from slam